pkg/test/options.go

```diff
@@ -51,7 +51,7 @@ func Options(overrides ...OptionsFields) *azoptions.Options {
 		KubeletClientTLSBootstrapToken: lo.FromPtrOr(options.KubeletClientTLSBootstrapToken, "test-token"),
 		SSHPublicKey:                   lo.FromPtrOr(options.SSHPublicKey, "test-ssh-public-key"),
 		NetworkPlugin:                  lo.FromPtrOr(options.NetworkPlugin, "azure"),
-		NetworkPolicy:                  lo.FromPtrOr(options.NetworkPolicy, ""),
+		NetworkPolicy:                  lo.FromPtrOr(options.NetworkPolicy, "cilium"),
```

Comment:
"
--network-policy
The Kubernetes network policy to use.

Using together with "azure" network plugin. Specify "azure" for Azure network policy manager, "calico" for calico network policy controller, "cilium" for Azure CNI powered by Cilium. Defaults to "" (network policy disabled).
"

The aks API you specify network policy as cilium so this better reflects what this should be

Reply:
just did what you said here

Reply:
Did not notice this was test; and elsewhere there was networkPolicyCilium used for dataplane label, so it seemed like you are trying to introduce proper support for specifying dataplane - which we likely already need, since there is a label we put? In any case, the default is "", the real E2E clusters we use (even with cilium dataplane) don't enable it (right?), and nothing actually depends or uses policy right now (we should probably remove it), so I don't quite see the point of changing it here. As long as you are looking for opportunities to improve something in passing - a better improvement would be to delete it. 🙂